default/python/AI/AIFleetMission.py

```diff
@@ -500,15 +512,15 @@ def issue_fleet_orders(self):
                         return  # colonize order must not have completed yet
                 clear_all = True
                 last_sys_target = INVALID_ID
+                secure_targets = set(
+                    AIstate.colonyTargetedSystemIDs
+                    + AIstate.outpostTargetedSystemIDs
+                    + AIstate.invasionTargetedSystemIDs
+                )
```

Comment:
I don't have old python installed.  This works in Pyhton3.9, but I am not sure if it works in 3.6

You can create set with unpacking variable.
```python
a = [1,2, 3]
b = [3,4, 5]
{*a, *b}
{1, 2, 3, 4, 5}
```

Reply:
Well, I just moved this code, because I needed the set outside if clause. There are surely more important things to refactor than this.